GitUI/UserControls/RevisionGrid/RevisionDataGridView.cs

```diff
@@ -36,6 +37,9 @@ public sealed partial class RevisionDataGridView : DataGridView
 
         private int _backgroundScrollTo;
         private int _rowHeight; // Height of elements in the cache. Is equal to the control's row height.
+        private int _mouseWheelDeltaRemainder; // Corresponds to unconsumed scroll distance while scrolling via mouse wheel, see OnMouseWheel().
+
+        private long _lastMouseWheelTickCount; // Timestamp of the last vertical scroll via mouse wheel.
```

Comment:
insert new fields alphabetically sorted and without empty lines, please

Reply:
Call me stupid... It's not clear to me, sorry. Without touching existing empty lines? Like this?

```
 ...
 private int _loadedToBeSelectedRevisionsCount = 0;

 private int _backgroundScrollTo;
 private long _lastMouseWheelTickCount;
 private int _mouseWheelDeltaRemainder;
 private int _rowHeight;

 private VisibleRowRange _visibleRowRange;
 ...
```

Or structured by association, like this?

```
 ...
 private int _loadedToBeSelectedRevisionsCount = 0;

 private int _backgroundScrollTo;
 private int _rowHeight;

 private long _lastMouseWheelTickCount;
 private int _mouseWheelDeltaRemainder;

 private VisibleRowRange _visibleRowRange;
 ...
```

Reply:
You are not at all. I did not want to request a sorting of all fields. So the first one might be the best compromise.
Structuring is arbitrary too often. "Insert alphabetically" is our simple coding style.